### PR TITLE
PP-6178 - Adding routing code to server correct apple certs for each environment

### PR DESCRIPTION
**Pull request by narinderchana**
Description:

We decided to use configuration and the express middleware to deliver this functionality as this make for minimal changes.
Frontend will server the correct certificate for the environment from the environment folders i.e production-2 will be served from /app/assets/apple-pay/production-2.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -129,6 +129,7 @@ function initialiseTemplateEngine (app) {
 }
 
 function initialisePublic (app) {
+  app.use('/.well-known/apple-developer-merchantid-domain-association.txt', express.static(path.join(__dirname, `/app/assets/apple-pay/${process.env.ENVIRONMENT}/apple-developer-merchantid-domain-association.txt`)))
```
Comment (alexbishop1) [inline server.js:132]: What happens if `ENVIRONMENT` isn’t set or is set to a value that doesn’t match one of the directories? Do we get a 404?

File: app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt [hunk 2]
```diff
@@ -1,72 +1,72 @@
...
+dXRob3JpdHkCCD1yIOPPjPIlMAkGBSsOAwIaBQCggdwwGAYJKoZIhvcNAQkDMQsGCSqGSIb3DQEH
+ATAcBgkqhkiG9w0BCQUxDxcNMjAwMjAzMTE1NjQ1WjAjBgkqhkiG9w0BCQQxFgQUeZa8p8PhNNHE
+gbQ4P3K8gQCftxUwKQYJKoZIhvcNAQk0MRwwGjAJBgUrDgMCGgUAoQ0GCSqGSIb3DQEBAQUAMFIG
+CSqGSIb3DQEJDzFFMEMwCgYIKoZIhvcNAwcwDgYIKoZIhvcNAwICAgCAMA0GCCqGSIb3DQMCAgFA
+MAcGBSsOAwIHMA0GCCqGSIb3DQMCAgEoMA0GCSqGSIb3DQEBAQUABIIBAHTh_C1wzInT3muqdMrb
+Hr6clXUR5ZVpf2UE1N1t0xJ-_9Qv9A9OanLVAgtrDO1GuwyNPqdhS47yIFL6w8MF1Z3-4LC9YAm8
+_l9rUkxEYiJa50TH8okH7pP6AQT1q_jl18lpsz6NiczWuGePDzKnwzlLm3CAmAFzpbRsYo_aUCDD
+5hJYDBfS_yQyScQOSJPmet6sEP80bWSkqf71QNsnVo7Fqw2CUY8O9d8f4mWe9l0o9Ko_2ynLaFj5
+1eR3zwnzgvRuSMsK2TWk8kMOLz87WGq721pFFLPV__e4F3UFK_MJoIuvKJtv8IIyZDEGWSJz_Ekl
+iajV91M9KSswwZgs6Wo
```
Comment (alexbishop1) [inline app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt:72]: Curious as to why this file has changed. We already re-verified test-12. Did we download the file again afterwards?

File: server.js (hunk 1 above)
Comment (nimalank7) [reply server.js:132]: Yep we get a 404

File: app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt (hunk 2 above)
Comment (narinderchana) [reply app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt:72]: I'm not sure how that actually changed, we did have a merge conflict. May be when resolved we picked up an old version. Let us fix that so it is the version you checked in last Thursday.

Comment (narinderchana) [reply app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt:72]: I have just checked the contents of stage/prod they are correct. Just this one need to be changed

Comment (narinderchana) [reply app/assets/apple-pay/test-12/apple-developer-merchantid-domain-association.txt]: We have now corrected the test file.

**Pull request merged**